src/constants/base-contracts.ts

```diff
@@ -61,7 +61,7 @@ pragma solidity ^0.8.0;
 
 import "@thirdweb-dev/contracts/base/ERC721DelayedReveal.sol";
 
-contract MyContract is ERC721DelayedReveal {
+contract Contract is ERC721DelayedReveal {

```

Comment:
how come we changed these - `MyContract` matches with the name we have in the template repos - did u change them?